quantus/helpers/explanation_func.py

```diff
@@ -1,22 +1,35 @@
 """This modules contains explainer functions which can be used in conjunction with the metrics in the library."""
 from typing import Union
+
+import numpy as np
 import torch
 import scipy
 import random
+from importlib import util
 import cv2
 from captum.attr import *
 import warnings
 from .utils import *
 from .normalise_func import *
 
 
+if not (util.find_spec("captum") or util.find_spec("tf_explain")):
+    raise ImportError(
+        "Explanation library not found. Please install Captum for torch models and tf-explain for TensorFlow."
+    )
+if util.find_spec("captum"):
+    from captum.attr import *
+if util.find_spec("tf_explain"):
+    import tf_explain
+
+
 def explain(
     model: torch.nn,
```

Comment:
typing of model, inputs and targets parameters is not up-to-date for tensorflow integration, right?